app/panel/components/BuildingBlocks/ModalExitButton.jsx

```diff
@@ -22,19 +23,30 @@ import PropTypes from 'prop-types';
  */
 const ModalExitButton = (props) => {
 	const {
-		toggleModal
+		toggleModal,
+		border
 	} = props;
 
+	const borderClassNames = ClassNames('ModalExitButton__exit flex-container align-middle', {
+		green: border === 'green'
+	});
+
```

Comment:
Instead of `green`, let's just apply the value of `border` - this also makes sure that `grey` is applied in the default case.

Reply:
Perfect